cli/js/lib.deno.ns.d.ts

```diff
@@ -1368,6 +1374,25 @@ declare namespace Deno {
    */
   export function shutdown(rid: number, how: ShutdownMode): void;
 
+  /** A socket is a generic transport listener for message-oriented protocols */
```

Comment:
Can you mark all of the new functions, classes, interfaces as `UNSTABLE: new API` in the JSDOC

You'll see examples of this elsewhere in this file. 

https://github.com/denoland/deno/blob/6bd846a780bec8a60d0a251ed1fb43e3add81be5/cli/js/lib.deno.ns.d.ts#L98